test/players.js

```diff
@@ -159,6 +160,83 @@ describe('GET /players/:id', () => {
         assert.strictEqual(seasonsInResponse.length, seasonsInService.seasons.length,
             'should return correct number of records');
     });
+
+    it.only('should return restructured object received by getById() method', async () => {
```

Comment:
please make sure you've removed things you use for debug before pushing your code